### DISCUSSION TITLE
Kernel 6.1 mglru

### DISCUSSION THREAD

**Discussion by Fak3** [hakavlad/nohang#124] [General]
Has anyone tried to test new Kernel's mglru? Does it prevent the hang and correctly invokes oom?

**Comment by hakavlad** [comment]
min_ttl_ms works well

**Reply by Fak3** [reply]
Now the question is how to get desktop notification when oom kills process🤔

**Reply by hakavlad** [reply]
Use nohang.

**Comment by Fak3** [comment]
Hmm... Recently, on kernel ~ 6.7 and later (6.8.8 now) i noticed that mglru does not help anymore, system hangs. It was surely working for me earlier, on kernels 6.1+
Anyone has same experience?

**Reply by hakavlad** [reply]
I not tested new kernels.
